rest/revocation_test.go

```diff
@@ -47,20 +47,20 @@ const (
 	revocationTestPassword = "test"
 )
 
-func (tester *ChannelRevocationTester) addRole(user, role string) {
+func (tester *ChannelRevocationTester) addRole(role string) {
```

Comment:
These changes I guess were pulled over from the previous branch/PR

https://github.com/couchbase/sync_gateway/pull/6554#discussion_r1373231754